charts/tekton-apps/templates/triggers.yaml

```diff
@@ -0,0 +1,83 @@
+{{- range $project := .Values.apps }}
+{{- if $project.enabled }}
+{{- range $component := $project.components }}
+
+{{/* initialize `projectEnvironment` variable with either `$project.environment` if it exists or with `$.Values.environment` otherwise */}}
+{{ $projectEnvironment := ternary $project.environment $.Values.environment (hasKey $project "environment") -}}
+
+{{/* initialize `gitBranchPrefixes` variable with either `$component.eventlistener.gitWebhookBranches` list if it exists or `$.Values.gitBranchPrefixes` otherwise */}}
+{{ $gitBranchPrefixes := ternary $component.eventlistener.gitWebhookBranches $.Values.gitBranchPrefixes (hasKey $component.eventlistener "gitWebhookBranches") -}}
+
+{{- $data := dict "project"           $project
```

Comment:
Let's switch `data` and `filter`, i.e. `filter` would go first, to separate dict variable from other ones?

Reply:
I'm not sure what's the difference, but ok.